src/components/Sweep/SweepExecute.tsx

```diff
@@ -0,0 +1,202 @@
+import { useCallback, useState, useEffect } from 'react';
+import { Form, Input, Button, notification, Typography } from 'antd';
+import { ethers } from 'ethers';
+import styled from 'styled-components';
+
+import { ethereumAddressValidator } from '../../utils';
+import { useMetamask } from '../../hooks/useMetamask';
+import { ChainId } from '../../constants';
+import { contractBalanceOf, contractTransferFrom, contractAllowance } from './useSweep';
+
+const { Text } = Typography;
+
+function SweepExecute() {
+  const [loading, setLoading] = useState(false);
+  const [valid, setValid] = useState(false);
+
+  const [fromAddress, setFromAddress] = useState('');
```

Comment:
why is this called `fromAddress` but below you have just `to` and not `toAddress`? you need to be consistent in your naming always